logging/apiv2/config_client.go

```diff
@@ -572,7 +572,7 @@ func (c *configGRPCClient) Connection() *grpc.ClientConn {
 // the `x-goog-api-client` header passed on each request. Intended for
 // use by Google-written clients.
 func (c *configGRPCClient) setGoogleClientInfo(keyval ...string) {
```

Comment:
Please revert files in this directory, they can only be updated by our regeneration job.

Reply:
would it be OK to replicate `versionGo()` function?